CMakeLists.txt

```diff
@@ -102,6 +102,28 @@ if (COMPILER_CLANG)
     # Clang doesn't have int128 predefined macros, workaround by manually defining them
     # Reference: https://stackoverflow.com/questions/41198673/uint128-t-not-working-with-clang-and-libstdc
     set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D__GLIBCXX_BITSIZE_INT_N_0=128 -D__GLIBCXX_TYPE_INT_N_0=__int128")
+
+    option(ENABLE_TIME_TRACES "Enable clang feature time traces" OFF)
```

Comment:
Are these options Clang-specific? If so, we should check if the compiler is Clang then set. Otherwise GCC will be broke - at least GCC 7.x is in the support list so far.

Reply:
Those options are clang specific. They are under `if (COMPILER_CLANG)`.